src/layouts/DashboardLayout.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import React, { Component } from 'react';
+import { Card } from 'react-bootstrap';
+export class DashboardLayout extends Component {
```

Comment:
Mind adding a newline between the imports and export?

Reply:
fixed